frontend/src/api/recipe.ts

```diff
@@ -49,4 +57,21 @@ export const submitRecipe = async (formData: FormData) => {
     return response
 }
 
+export const submitComment = async (
+    formData: FormData,
+    recipeId: string = ''
```

Comment:
In TypeScript, when you declare a parameter with a default value (like recipeId: string = ''), it makes the parameter optional in terms of function invocation, even though the syntax doesn't include a ? mark.

Why using a default value? This parameter is required. Removing it ensures you to use the function properly when ivoking it